bigtable/client/cell.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/cell.h"
+
+#include <absl/strings/str_join.h>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+// consolidate concatenates all the chunks and caches the resulting value. It is
```

Comment:
That comment belongs in the header, as part of the Doxygen block so it is exported for our users to see.

Reply:
done.